### PR TITLE
Add Kubeflow 1.9 Release page

### PR DESCRIPTION
**Pull request by rimolive**

### CODE REVIEW

**Review by andreyvelich — COMMENTED** [review]
Thank you @rimolive!
/lgtm
/assign @terrytangyuan @johnugeorge @jbottum @james-jwu

File: content/en/docs/releases/kubeflow-1.9.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+      </td>
+    </tr>
+    <tr>
+      <th class="table-light">Media</th>
+      <td>
+        <b>Blog:</b> 
+          <a href="https://blog.kubeflow.org/kubeflow-1.9-release/">Kubeflow 1.9 Release Announcement</a>
+        <br>
+        <b>Video:</b> 
+          <a href="https://www.youtube.com/watch?v=CUQT-YccpR8">Kubeflow 1.9 Release Overview</a>
```
Comment (thesuperzapper) [inline content/en/docs/releases/kubeflow-1.9.md:25]: This video is for Kubeflow 1.7.

**Review by thesuperzapper — CHANGES_REQUESTED** [review]

File: content/en/docs/releases/kubeflow-1.9.md (hunk 1 above)
Comment (rimolive) [reply content/en/docs/releases/kubeflow-1.9.md]: Changed the text to `To be published`.

**Comment by rimolive** [comment]
@andreyvelich @james-jwu @johnugeorge @terrytangyuan @jbottum @thesuperzapper This PR is blocking the Kubeflow 1.9 blog post to be published. Please help with approving this asap.

**Review by terrytangyuan — APPROVED** [review]
/lgtm
/approve

**Pull request merged**